homeassistant/components/starlink/binary_sensor.py

```diff
@@ -114,4 +114,10 @@ def is_on(self) -> bool | None:
         entity_category=EntityCategory.DIAGNOSTIC,
         value_fn=lambda data: data.alert["alert_unexpected_location"],
     ),
+    StarlinkBinarySensorEntityDescription(
+        key="connected",
+        translation_key="connected",
```

Comment:
This doesn't need a translation as it gets one from its device class

Reply:
I thought this was the translation key for the name of the sensor, not the value?

Reply:
exactly. But if you set it to `None` (or don't set it) and it has a device class, it will take the default translation for the name  (and icon) from the device class

Reply:
Done, removed `translation_key` and the corresponding key from strings.